web/templates/inbox_view/inbox_row.hbs

```diff
@@ -48,19 +48,19 @@
-                          data-stream-id="{{stream_id}}" data-topic-name="{{topic_name}}" tabindex="0" data-col-index="{{ column_indexes.TOPIC_VISIBILITY }}">
+                          data-stream-id="{{stream_id}}" data-stream-name="{{stream_name}}" data-topic-name="{{topic_name}}" tabindex="0" data-col-index="{{ column_indexes.TOPIC_VISIBILITY }}">
                             {{#if (eq visibility_policy all_visibility_policies.FOLLOWED)}}
-                                <i class="zulip-icon zulip-icon-follow recipient_bar_icon" data-tippy-content="{{t 'You follow this topic'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'You follow this topic' }}"></i>
+                                <i class="zulip-icon zulip-icon-follow recipient_bar_icon" data-tippy-content="{{t 'You follow this topic.'}}"
+                                  role="button" aria-haspopup="true" aria-label="{{t 'You follow this topic.' }}"></i>
                             {{else if (eq visibility_policy all_visibility_policies.UNMUTED)}}
-                                <i class="zulip-icon zulip-icon-unmute-new recipient_bar_icon" data-tippy-content="{{t 'You have unmuted this topic'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'You have unmuted this topic' }}"></i>
+                                <i class="zulip-icon zulip-icon-unmute-new recipient_bar_icon" data-tippy-content="{{t 'You have unmuted this topic.'}}"
+                                  role="button" aria-haspopup="true" aria-label="{{t 'You have unmuted this topic.' }}"></i>
                             {{else if (eq visibility_policy all_visibility_policies.MUTED)}}
-                                <i class="zulip-icon zulip-icon-mute-new recipient_bar_icon" data-tippy-content="{{t 'You have muted this topic'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'You have muted this topic' }}"></i>
+                                <i class="zulip-icon zulip-icon-mute-new recipient_bar_icon" data-tippy-content="{{t 'You have muted this topic.'}}"
+                                  role="button" aria-haspopup="true" aria-label="{{t 'You have muted this topic.' }}"></i>
                             {{else if (eq visibility_policy all_visibility_policies.INHERIT)}}
-                                <i class="zulip-icon zulip-icon-inherit recipient_bar_icon" data-tippy-content="{{t 'Default'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'Default' }}"></i>
+                                <i class="zulip-icon zulip-icon-inherit recipient_bar_icon" data-tippy-content="{{t 'Notifications are based on your configuration for'}}"
+                                  role="button" aria-haspopup="true" aria-label="{{t 'Notifications are based on your configuration for' }}" privacy-policy="{{privacy_policy}}"></i>
```

Comment:
Here the aria-label would be "Notifications are based on your configuration for", I think you missed something here?